hotshot-types/src/epoch_membership.rs

```diff
@@ -179,6 +180,23 @@ where
             .await
             .ok_or(anytrace::warn!("add epoch root failed"))?;
         updater(&mut *(self.membership.write().await));
+
+        let drb_membership = match root_membership.next_epoch_stake_table().await {
+            Ok(drb_membership) => drb_membership,
+            Err(_) => Box::pin(self.wait_for_catchup(root_epoch + 1)).await?,
+        };
+
+        // get the DRB from the last block of the epoch right before the one we're catching up to
+        let Ok(drb) = drb_membership
+            .get_epoch_drb(transition_block_for_epoch(
+                *(root_epoch + 1),
+                self.epoch_height,
+            ))
+            .await
+        else {
+            anytrace::bail!("get epoch drb failed for in epoch {:?}", root_epoch + 1);
```

Comment:
```suggestion
            return anytrace::Err(anytrace::warn!("get epoch drb failed for in epoch {:?}", root_epoch + 1));
```

just to be explicit about the level